testing/tests/DevExpress.ui.widgets.editors/dateRangeBox.tests.js

```diff
@@ -3893,3 +3894,164 @@ QUnit.module('calendarOptions', moduleConfig, () => {
         });
     });
 });
+
+QUnit.module('Aria accessibility', moduleConfig, () => {
+    QUnit.test('aria-owns attribute on root element', function(assert) {
```

Comment:
1. We write about "root element" in tests, but there are also asserts for inputs
2. What we expect in this test? aria-owns should be always undefined?